lib/converters.js

```diff
@@ -25,6 +25,22 @@ function indent(content, spaces, ignoreFirst) {
     }).join("\n");
 }
 
+function emphasisFormats( text ) {
+  return {
+    strong: '**',
```

Comment:
@mitogh I have no experience in using such configuration files for how the syntax should look like.  Any ideas you have, please comment in https://github.com/IonicaBizau/code-style/issues/4. There is a similar discussion. I need help in writing such a file. :smile: